client/components/checklist/style.scss

```diff
@@ -57,6 +57,7 @@ $chevron-animation: transform 0.15s cubic-bezier( 0.175, 0.885, 0.32, 1.275 );
 		border: 2px solid var( --color-neutral-5 );
 		border-radius: 16px;
 		background: var( --color-surface );
+		z-index: 2;
```

Comment:
Would it be better to use the Sass z-index function instead? I *think* that's the recommended way: https://wpcalypso.wordpress.com/devdocs/docs/coding-guidelines/css.md#z-index